### PR TITLE
Fixing typo for issue #3247

### PR DESCRIPTION
**Pull request by Abhijnan-Bajpai**

### CODE REVIEW

**Comment by withshubh** [comment]
Hey @Abhijnan-Bajpai 
- Please change the branch from `ToolJet:main` to `ToolJet:develop`
- Please sign the CLA

**Review by withshubh — CHANGES_REQUESTED** [review]
Hey @Abhijnan-Bajpai 
- Please change the branch from `ToolJet:main` to `ToolJet:develop`
- Please sign the CLA

**Comment by Abhijnan-Bajpai** [comment]
> Hey @Abhijnan-Bajpai
> 
> * Please change the branch from `ToolJet:main` to `ToolJet:develop`
> * Please sign the CLA

Done

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-1.16.0
+1.16.1
```
Comment (withshubh) [inline .version:1]: Please undo the changes made in both the `.version` files

**Review by withshubh — CHANGES_REQUESTED** [review]

File: .version (hunk 1 above)
Comment (Abhijnan-Bajpai) [reply .version]: Check now?

**Review by withshubh — APPROVED** [review]
